src/server/js/schema.js

```diff
@@ -0,0 +1,14 @@
+import { merge } from 'lodash';
+import { makeExecutableSchema } from 'graphql-tools';
+
+import {
+    typeDefs as Health,
+    resolvers as healthResolvers,
+} from './resources/health/schema';
```

Comment:
this file is the root schema. as we implement actual models, they will replace this mock health check schema here